src/Exception/OAuthException.php

```diff
@@ -1,14 +1,14 @@
 <?php
 
+declare(strict_types=1);
+
 namespace BenjaminFavre\OAuthHttpClient\Exception;
 
 use RuntimeException;
 
 /**
  * Represents errors during OAuth protocol.
- *
- * @author Benjamin Favre <favre.benjamin@gmail.com>
```

Comment:
I see that the coding standard removes the `@author` phpdoc. Do you want me to skip/remove that rule in the config?